scripts/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     logging:
       driver: syslog
   orderly_server:
-    image: docker.montagu.dide.ic.ac.uk:5000/orderly.server:${MONTAGU_ORDERLY_SERVER_VERSION}
+    image: vimc/orderly.server:${MONTAGU_ORDERLY_SERVER_VERSION}
```

Comment:
The 'can run report' selenium test was also failing on TeamCity because orderly.server needs to come from the public registry now.